extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -601,6 +609,30 @@ class AmpAccordion extends AMP.BaseElement {
       tryFocus(newFocusHeader);
     }
   }
+
+  /**
+   * Callback function to execute when mutations are observed.
+   * @param {!Array<!MutationRecord>} mutations
+   */
+  toggleMutations_(mutations) {
+    if (mutations.length > 1) {
+      return;
+    }
+    const mutation = mutations[0];
+    if (mutation.attributeName !== 'expanded') {
+      return;
+    }
+    const sectionEl = dev().assertElement(mutation.target);
+    if (!sectionEl.hasAttribute('[expanded]')) {
+      return;
+    }
+    const toExpand = sectionEl.getAttribute('expanded') !== 'collapse';
+    if (toExpand) {
```

Comment:
I really do not like this API, we should really just create another attribute that controls whether or not the component is requested to be expanded. For example, `expand` and have `expanded` track whether or not it is actually expanded.

Reply:
I don't understand, can you help me? I thought that the `expanded` and `[expanded]` API was the same as any other bindable state attribute in other AMP components. Like `hidden` and `[hidden]`. Can you state your reasons for preferring separate `expand` and `expanded` attributes? It seems like it would be hard to explain to users why this is different than other components.